### PR TITLE
GEOPY-1190

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-1190 - MT forward simulation does not match 0.10.0**

### CODE REVIEW

File: geoapps/inversion/components/topography.py [hunk 1]
```diff
@@ -96,7 +97,23 @@ def active_cells(self, mesh: InversionMesh, data: InversionData) -> np.ndarray:
             print(
                 "Adjusting active cells so that receivers are all within an active cell . . ."
             )
-            active_cells[get_containing_cells(mesh.mesh, data)] = True
+            containing_cells = get_containing_cells(mesh.mesh, data)
```
Comment (benk-mira) [inline geoapps/inversion/components/topography.py:100]: This function is starting to get too complicated.  Can we pull out your additions to a new function?  Also, some common code we can pull out:
```
grid_reference = "bottom" if self.params.inversion_type in forced_to_surface else center"
active_cells = active_from_xyz(
    mesh.entity, self.locations, grid_reference=grid_reference
)
active_cells = active_cells[np.argsort(mesh.permutation)]
if self.params.inversion_type in forced_to_surface:
    ...
    if isinstance(mesh.mesh, TreeMesh):
        self._ensure_tree_mesh_connectivity()
```

Comment (benk-mira) [reply geoapps/inversion/components/topography.py:100]: ...then drop the bottom else statement

**Review by benk-mira — DISMISSED** [review]
Requesting a small refactor. I'll leave it up to you if it's worth doing now

File: geoapps/inversion/components/topography.py (hunk 1 above)
Comment (domfournier) [reply geoapps/inversion/components/topography.py:100]: Good idea. Done.

File: geoapps/inversion/components/topography.py [hunk 2]
```diff
@@ -81,34 +82,28 @@ def active_cells(self, mesh: InversionMesh, data: InversionData) -> np.ndarray:
...
-            )
-            active_cells = active_cells[np.argsort(mesh.permutation)]
-            print(
-                "Adjusting active cells so that receivers are all within an active cell . . ."
-            )
-            active_cells[get_containing_cells(mesh.mesh, data)] = True
+        active_cells = active_from_xyz(
+            mesh.entity,
+            self.locations,
+            grid_reference="bottom" if forced_to_surface else "center",
```
Comment (benk-mira) [inline geoapps/inversion/components/topography.py:95]: even better!

File: geoapps/inversion/components/topography.py [hunk 3]
```diff
@@ -136,3 +131,35 @@ def get_locations(self, obj: Entity) -> np.ndarray:
                 locs[:, 2] = elev
 
         return locs
+
+    def expand_actives(
```
Comment (benk-mira) [inline geoapps/inversion/components/topography.py:135]: rad

**Review by benk-mira — APPROVED** [review]
Thanks for the updates, looks great!

**Pull request merged**